package.json

```diff
@@ -20,7 +20,8 @@
         "core-js-builder": "^3.36.1",
         "ecmarkup": "^18.3.1",
         "jest": "^29.7.0",
-        "prettier": "^3.2.5"
+        "prettier": "^3.2.5",
+        "typescript": "^5.7.2"
```

Comment:
I think we don't need to pin the TypeScript version because it isn't published to npm.